src/golang/lib/repos/sqlite/operator.go

```diff
@@ -84,6 +97,23 @@ func (*operatorReader) GetByDAG(ctx context.Context, dagID uuid.UUID, DB databas
 	return getOperators(ctx, DB, query, args...)
 }
 
+func (*operatorReader) GetNodesByDAG(
+	ctx context.Context,
+	dagID uuid.UUID,
+	DB database.Database,
+) ([]views.OperatorNode, error) {
+	query := fmt.Sprintf(
```

Comment:
This is where I think we should put the query. We can also get rid of `views.OperatorNodeCols()` since the columns are specific to this query only.

Reply:
It's reused in `GetNode()` so I think some variables should be helpful here. Also since `opView` and `artfView` follows how node objects are defined in API endpoints, it makes sense to assume we will have more queries on these views.

But I agree we should move subquery here as it's language specific and is a part of query implementation